lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/util/YarnTypes.java

```diff
@@ -33,6 +33,9 @@
 @Private
 public final class YarnTypes {
 
+  // TODO: this is currently being developed on a hacked 2.4.0 bits, should be 2.4.1
+  public static final String MIN_VERSION_KEEP_CONTAINERS_AVAILABLE = "2.4.0";
```

Comment:
This can be removed once [REEF-573](https://issues.apache.org/jira/browse/REEF-573) is resolved, right? Maybe Mark the TODO as such: `// TODO[REEF-573]: Remove once the Hadoop version is updated.`